sdk/sdk.go

```
@@ -240,7 +240,8 @@ func (babylonClient *BabylonQueryClient) QueryIsBlockBabylonFinalized(queryParam
 
 	// no FP has voting power for the consumer chain
 	if totalPower == 0 {
-		return false, nil
+		// skip the block if no FP has voting power
+		return true, nil
```

wouldn't this mix the facts of a block is truely finalized or no voting power? Maybe we can return a concrete error type to let the caller decide?

---

agree that it's better not to fix

---

It makes sense, I will define an `error` type and return it.